types/node/test/wasi.ts

```diff
@@ -12,8 +12,10 @@ import * as fs from 'node:fs';
     const importObject = { wasi_snapshot_preview1: wasi.wasiImport };
 
     (async () => {
-        const wasm = await WebAssembly.compile(fs.readFileSync('./demo.wasm'));
-        const instance = await WebAssembly.instantiate(wasm, importObject);
+        // TODO: WebAssembly types don't exist in @types/node
+        // const wasm = await WebAssembly.compile(fs.readFileSync('./demo.wasm'));
+        // const instance = await WebAssembly.instantiate(wasm, importObject);
+        const instance = {};
 
         wasi.start(instance);
```

Comment:
This seems ugly. But `WebAssembly` in lib.dom.d.ts is _not_ compatible with NodeJS (extra methods) so this seems like a possible solution until `WebAssembly` officially is typed in `@types/node`. (`was.start()` is still type checked here: it simply accepts type `object`)